platform/harmony/transaction.go

```diff
@@ -8,6 +8,8 @@ import (
 	"strconv"
 )
 
+const Annual = 10
```

Comment:
Is any better way to calculate APR ?

Reply:
This is only used as fallback Apr if there's no validator available.

The APR is calculated  line 43 of platform/harmony/stake.go,  we go through all returned active validators and get the validator with maximal APR and use it as the current APR. 

https://github.com/trustwallet/blockatlas/pull/1099/files#diff-c861e736adc338bed48219a604ff84b7R43

func (p *Platform) GetMaxAPR() float64 {
	validators, err := p.client.GetValidators()
	if err != nil {
		logger.Error("GetMaxAPR", logger.Params{"details": err, "platform": p.Coin().Symbol})
		return Annual
	}

	var max = 0.0
	for _, e := range validators.Validators {
		var apr float64
		if apr, err = strconv.ParseFloat(e.Lifetime.Apr, 64); err != nil {
			apr = 0.0
		}

		if apr > max {
			max = apr
		}
	}

	return max
}